lambda.js

```diff
@@ -22,9 +28,15 @@ const actions = {
       };
     },
     get: async (name, ref) => {
+      if (ref.indexOf("sha256") !== -1) {
```

Comment:
Could you make this a bit more robust in case someone names a tag `sha256something` this would break.
Along the lines of 

```js
// untested
ref.indexOf("sha256:") === 0 && ref.match(/^sha256:[a-f0-9]+$/)
```

Reply:
good point, I'll see what works best